### PR TITLE
create userplans automatically

### PR DESCRIPTION
**Pull request by PetrDlouhy**
This addresses issue #43 by various methods. It creates `UserPlan`s during initial migration, by management command and it allows to set user in UserPlanAdmin (not removing the link).

### CODE REVIEW

**Comment by PetrDlouhy** [comment]
Note: Both PRs #80 and #79 contains migrations which would conflict if pulled together. So please pull one of the PRs first and I will make the appropriate migration changes for the second before pulling.

**Comment by PetrDlouhy** [comment]
@Alir3z4 I have added changelog record to this and changed the `list_display` back to tuple as requested in #82

**Comment by Alir3z4** [comment]
@PetrDlouhy should we close this pull request if the it has been addressed in other pull requests ?

**Comment by PetrDlouhy** [comment]
@Alir3z4 I don't think, the changes are included in other PRs, or they should not. I tried to make separate PR for every problem, but they are clashing with migrations and changelog changes.

**Comment by Alir3z4** [comment]
Ok, the pull request looks good to me.
However it has code that are not covered in the tests.
You can see the uncovered code at https://codecov.io/gh/django-getpaid/django-plans/pull/80/diff

Let's write tests to cover those parts and we can merge it.

**Comment by PetrDlouhy** [comment]
@Alir3z4 OK, I will do that.

**Comment by Alir3z4** [comment]
Great!

Let me know when it's ready so I can take a look ;)

**Comment by PetrDlouhy** [comment]
I have created the test, but I can't find the Codecov report. Anyway the `create_for_users_without_plan` and `create_userplans` management command should be covered.

**Comment by Alir3z4** [comment]
coverage won't be able to catch that, because call_command spin another process to run the management command.

However, still we need to somehow test the output of that command.
I see there's no output from the command, either failure or success.

I recommend to print out something once the management done, something simple like "User plans created " or anything more you find meaningful, then you can catch that from stdout within the tests.

This is how you'd be writing tests for stdout when calling a management command: https://docs.djangoproject.com/en/dev/topics/testing/tools/#management-commands

File: plans/management/commands/create_userplans.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.core.management import BaseCommand
+
+from plans.models import UserPlan
+
+
+class Command(BaseCommand):
+    help = 'Creates UserPlans for all Users'
+
+    def handle(self, *args, **options):
```
Comment (Alir3z4) [inline plans/management/commands/create_userplans.py]: Once you have done the as suggested in https://github.com/django-getpaid/django-plans/pull/80#issuecomment-431608635 you can exclude this from coverage via `#  pragma: no cover` comment.

```suggestion
    def handle(self, *args, **options):  # pragma: no cover
```

**Comment by Alir3z4** [comment]
@PetrDlouhy could you please check this pull request so I can merge ?

**Comment by PetrDlouhy** [comment]
@Alir3z4 I added direct test, and modified the command test by Your requirements. I also rebased this to fixed master (please merge #86 first) and added reverse code for the migration.

**Pull request merged**